app/controllers/search_tenancies_controller.rb

```diff
@@ -17,6 +17,8 @@ def show
       post_code: @post_code,
       tenancy_ref: @tenancy_ref
     )
+    @number_of_pages = @results.dig(:number_of_pages)
+    @tenancies = Kaminari.paginate_array(@results.dig(:tenancies)).page(params[:page]).per(20)
```

Comment:
Is this worth splitting up into multiple lines? I found it hard to parse.

```suggestion
    tenancies = @results.dig(:tenancies)
    page = params[:page]
    @tenancies = Kaminari.paginate_array(tenancies).page().per(20)
```

Reply:
I feel it reads fairly well now, and I don't want to clutter the controller with more variables.